app/models/message.rb

```diff
@@ -57,14 +57,14 @@ def total_upload_size
   end
 
   def total_upload_size_validation
-    return unless (total_upload_size.to_f / MEGABYTE) > 6.0
-    errors.add(:base, "Total size of uploads exceeds #{MAX_TOTAL_FILE_SIZE} MB")
+    return unless total_upload_size > MAX_TOTAL_FILE_SIZE_MB.megabytes
+    errors.add(:base, "Total size of uploads exceeds #{MAX_TOTAL_FILE_SIZE_MB} MB")
   end
 
   def each_upload_size_validation
     uploads.each do |upload|
-      next if (upload.size.to_f / MEGABYTE) <= MAX_SINGLE_FILE_SIZE
-      errors.add(:base, "#{upload.original_filename} exceeds file size limit of #{MAX_SINGLE_FILE_SIZE} MB")
+      next if upload.size <= MAX_SINGLE_FILE_SIZE_MB.megabytes
+      errors.add(:base, "The #{upload.original_filename} exceeds file size limit of #{MAX_SINGLE_FILE_SIZE_MB} MB")
```

Comment:
I think the text you had before makes more sense
'The upload.pdf exceeds file size limit' sounds strange compared to
'upload.pdf exceeds file size limit'